lib/manager.js

```diff
@@ -14,7 +14,7 @@ module.exports = class AssetManager {
 		if(manifestConfig) {
 			let manifestPath = this.resolvePath(manifestConfig.file,
 					{ enforceRelative: true });
-			this.manifest = new Manifest(manifestPath, manifestConfig);
+			this.manifest = new Manifest(manifestPath, manifestConfig, this);
```

Comment:
This feels wrong somehow...

Reply:
Why? We need to resolve a path inside the manifest. We therefore need the asset manager.

Reply:
`AssetManager` instantiates `Manifest` while passing itself; too circular for my taste. [Elsewhere](https://github.com/faucet-pipeline/faucet-pipeline-js/blob/34a31636c2bc8c845886f3b5ea8a92ddecff2093/lib/index.js#L10) I resorted to only passing `#resolvePath` as a function (which is why that's a _bound_ method), that avoids awareness of `AssetManager`.